### PR TITLE
Readme and spelling boost

### PR DESCRIPTION
**Pull request by TheJJ**
Mainly includes the new reddit link :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,17 +38,18 @@ Our goals *include*:
...
 * AI scripting in Python, you can use [machine learning](http://scikit-learn.org/stable/)
 * Re-creating [free game assets](https://github.com/SFTtech/openage-data)
 * An easily-moddable content format: **nyan**'s yet another notation
 * A powerful integrated Python console and interface, comparable to [blender](http://blender.org/)
 
-Our goals specifically *exclude*:
+Things we *won't* do:
 
-* Network compatibility with the original game.
+* No network compatibility with the original game.
```
Comment (franciscod) [inline README.md]: double negation here... "wont do no network compat"

File: README.md [hunk 2]
```diff
@@ -36,17 +38,18 @@ Our goals *include*:
...
   You really wanna have the same problems again?
-* Binary compatibility with the original game.
+* No binary compatibility with the original game.
```
Comment (franciscod) [inline README.md]: double negation here too

File: README.md [hunk 3]
```diff
@@ -36,17 +38,18 @@ Our goals *include*:
...
 * Multiplayer (obviously)
-* Optionally, [Improvements](/doc/ideas/) over the original game
+* Matchmaking and ranking with a [haskell masterserver](https://github.com/SFTtech/openage-masterserver)
+* Optionally, [improvements](/doc/ideas/) over the original game
 * AI scripting in Python, you can use [machine learning](http://scikit-learn.org/stable/)
 * Re-creating [free game assets](https://github.com/SFTtech/openage-data)
 * An easily-moddable content format: **nyan**'s yet another notation
 * A powerful integrated Python console and interface, comparable to [blender](http://blender.org/)
 
-Our goals specifically *exclude*:
```
Comment (franciscod) [inline README.md]: I liked this old one

**Comment by franciscod** [comment]
great!

File: README.md (hunk 3 above)
Comment (TheJJ) [inline README.md]: The thing is i wanted to formulate the actual content in a negative way, so it's clear even when read standalone. One newssite even misinterpreted it as "yes we'll do network compat"..
That's why I wanted to clarify it.

**Comment by TheJJ** [comment]
It should be fixed now, especially the double negations.

**Pull request merged**